prov/efa/src/efa_ep.c

```diff
@@ -251,6 +251,12 @@ static int efa_ep_close(fid_t fid)
 
 	ep = container_of(fid, struct efa_ep, util_ep.ep_fid.fid);
 
+	if (ep->scq)
+		free(ep->scq);
+
+	if (ep->rcq)
+		free(ep->rcq);
```

Comment:
Need to free after malloc in `efa_ep_bind_rdm`

Reply:
I do not think you want to call `free` here. This function is used for both rdm and dgram, and for dgram `ep-scq` and `ep->rcq` is not allocated, so should not be freed. I am surprised that `fi_dgram_pingpong` did not fail.

Reply:
fixed